src/plugins/yandex-metrika/types.ts

```diff
@@ -20,6 +20,10 @@ export interface MetrikaCounter {
     ecommerce?: boolean | string;
     /** @default 0 */
     type?: number;
+    /** @default {} */
+    userParams?: {
+        [x: string]: boolean | string | number;
+    };
```

Comment:
Is it full type https://yandex.ru/support/metrica/data/user-params_data.html?lang=en#online? As I see it there should be something like this

```ts
type UserParams = {
    [x: string]: boolean | string | number | null | UserParams;   
}
```

Reply:
update